### PR TITLE
Fixed all chefstyling and update configuration

### PR DESCRIPTION
**Pull request by piyushawasthi**
Signed-off-by: piyushawasthi <piyush.awasthi@msystechnologies.com>

### Description

[Please describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All tests pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,35 +1,6 @@
...
-  Enabled: false
-Layout/AlignHash:
-  Enabled: true
-  EnforcedLastArgumentHashStyle: ignore_implicit
-Layout/EndOfLine:
-  Enabled: false
+# AllCops:
+#   Exclude:
+#     - "spec/data/**/*"
+#     - "vendor/**/*"
```
Comment (tas50) [inline .rubocop.yml]: This file can get deleted

File: lib/win32/certstore/mixin/shell_out.rb [hunk 2]
```diff
@@ -69,7 +69,7 @@ def powershell_out!(*command_args)
         def run_command_with_os_architecture(script, options)
           options ||= {}
           options = options.dup
-          arch = options.delete(:architecture)
+          # arch = options.delete(:architecture)
```
Comment (tas50) [inline lib/win32/certstore/mixin/shell_out.rb]: If we’re not using this variable lets just remove it

**Comment by piyushawasthi** [comment]
@tas50 : Thanks for review, One more thing can we add appveyor badge in README.

**Comment by tas50** [comment]
Looks good.

**Pull request merged**